docs/source/api/v5/deliveryservices_id_servers.rst

```diff
@@ -40,6 +40,9 @@ Request Structure
 
 Response Structure
 ------------------
+:asns:           The :abbr:`ASN (Autonomous System Number)` associated with the cachegroups of the current server.
+
+	.. versionadded:: 4.1
```

Comment:
This is native to APIv5, you don't need to let people know what minor version of a previous API added it